test/verify/check-journal

```diff
@@ -458,6 +457,15 @@ s.send(b"PRIORITY=3\\nFOO=bar\\n")'
         b = self.browser
         m = self.machine
 
+        # We restart the reportd service,
+        # which causes the D-Bus proxy to spit out an error when it tries to
+        # query its managed objects.
+        self.allow_journal_messages('.*Remote peer disconnected')
+
+        m.upload(["verify/files/mock-bugzilla-server.py"], "/tmp/")
+        m.execute("setsid /tmp/mock-bugzilla-server.py >/tmp/mock-bugzilla.log 2>&1 &")
```

Comment:
There's `m.spawn()` for this, which gives you a pid back. [Example](https://github.com/cockpit-project/cockpit/blob/master/test/verify/check-metrics#L106)

Reply:
I’m going to assume that you are referring to the `timeout` hack for killing `sleep`.

Reply:
No, I mean that `m.execute("setsid ...")` is better done with `m.spawn("...")`. But it's not that important, I was just pointing out that this exists. The crashing sleep is fine.